packages/loader/lib/index.js

```diff
@@ -56,57 +76,21 @@ mainContext = this; // eslint-disable-line no-undef
     return exports;
   }
 
-  let isNode =
-    typeof window === 'undefined' &&
-    typeof process !== 'undefined' &&
-    {}.toString.call(process) === '[object process]';
-
-  if (!isNode) {
-    Ember = this.Ember = this.Ember || {};
-  }
-
-  if (typeof Ember === 'undefined') {
-    Ember = {};
-  }
-
-  if (typeof Ember.__loader === 'undefined') {
-    registry = Object.create(null);
-    seen = Object.create(null);
-
-    define = function (name, deps, callback) {
-      let value = {};
+  require = function (name) {
+    return internalRequire(name, null);
+  };
 
-      if (!callback) {
-        value.deps = [];
-        value.callback = deps;
-      } else {
-        value.deps = deps;
-        value.callback = callback;
-      }
+  // eslint-disable-next-line no-unused-vars
+  define = function (name, deps, callback) {
+    registry[name] = { deps: deps, callback: callback };
+  };
 
-      registry[name] = value;
-    };
+  // setup `require` module
+  require['default'] = require;
 
-    require = function (name) {
-      return internalRequire(name, null);
-    };
+  require.has = function registryHas(moduleName) {
+    return Boolean(registry[moduleName]) || Boolean(registry[moduleName + '/index']);
+  };
 
-    // setup `require` module
-    require['default'] = require;
-
-    require.has = function registryHas(moduleName) {
-      return Boolean(registry[moduleName]) || Boolean(registry[moduleName + '/index']);
-    };
-
-    require._eak_seen = registry;
-
-    Ember.__loader = {
-      define: define,
-      require: require,
-      registry: registry,
-    };
-  } else {
-    define = Ember.__loader.define;
-    require = Ember.__loader.require;
-  }
+  require._eak_seen = registry;
```

Comment:
```suggestion
  require.entries = registry;
```

https://github.com/ember-cli/loader.js/blob/ccd773bae5153b070f861a44d556d79446db8ac5/lib/loader/loader.js#L331